### PR TITLE
Trista frontend unique pair

### PR DESCRIPTION
**Pull request by sitianqiu**
In this PR,  I changed PersonalScheduleCreatePage.js and PersonalScheduleCreatePageTests.js, so when you try to store a personal schedule that matches the same quarter and name as another, you get a reasonable error message and a chance to fix the problem.
image

### CODE REVIEW

**Review by zackglazewski — CHANGES_REQUESTED** [review]
I wonder if you should remove .DS_Store

It must not be included in the .gitignore file and I'm pretty sure it is a file created by a Mac when you open a file via Finder. I also tested on the qa deployment and it works for me as it is in your example pictures. 

Once you get rid of this .DS_Store file, I will approve!

**Comment by sitianqiu** [comment]
> I wonder if you should remove .DS_Store
> 
> It must not be included in the .gitignore file and I'm pretty sure it is a file created by a Mac when you open a file via Finder. I also tested on the qa deployment and it works for me as it is in your example pictures.
> 
> Once you get rid of this .DS_Store file, I will approve!

I see. Working on it!

**Review by zackglazewski — DISMISSED** [review]
LGTM!

File: frontend/src/main/pages/PersonalSchedules/PersonalSchedulesCreatePage.js [hunk 1]
```diff
@@ -20,9 +20,14 @@ export default function PersonalSchedulesCreatePage() {
     toast(`New personalSchedule Created - id: ${personalSchedule.id} name: ${personalSchedule.name}`);
   }
 
+  const onError = (error) => {
+    // toast(`Error: ${error}`);
```
Comment (pconrad) [inline frontend/src/main/pages/PersonalSchedules/PersonalSchedulesCreatePage.js:24]: Can we remove this commented out line of code?

**Review by pconrad — CHANGES_REQUESTED** [review]
One line of commented out code to remove.. otherwise LGTM.

File: frontend/src/main/pages/PersonalSchedules/PersonalSchedulesCreatePage.js (hunk 1 above)
Comment (sitianqiu) [reply frontend/src/main/pages/PersonalSchedules/PersonalSchedulesCreatePage.js:24]: Sorry I missed this. I will fix it right now.

**Pull request merged**